geps/gep-3875/index.md

```diff
@@ -0,0 +1,192 @@
+  // Kind is kind of the resource.
+  Kind Kind `json:"kind"`
+  // Name is the name of the resource.
+  Name ObjectName `json:"name"`
+}
+```
+
+An example usage, to attach a BackendTLSPolicy for calls from Gateway `my-gateway` when calling the `app` Service in another namespace:
+
+```yaml
+kind: BackendTLSPolicy
+metadata:
+  name: app-pol
+  namespace: gateway-ns
+targetRef:
+  namespace: app-namespace
+  name: app
+  from:
+    kind: Gateway
+    name: my-gateway
+```
+
+An example, doing the same but for a specific HTTPRoute:
+
+```yaml
+kind: BackendTLSPolicy
+metadata:
+  name: app-pol
+  namespace: gateway-ns
+targetRef:
+  namespace: app-namespace
+  name: app
+  from:
+    kind: HTTPRoute
+    name: my-route
+```
+
+Including `SectionName` is also a likely addition that could be added for fine-grained control.
+
+The following `from` types will be `Core`:
+
+* Gateway
+* ListenerSet
+* All Route types where TLS is applicable (not UDPRoute)
+
+Implementations may have implementation specific `from` types as well.
+
+The inclusion of 'From' introduces a more complex hierarchy of policies.
+In the event multiple policies could apply to a certain traffic path, the most precise one will win.
+This means, in order of highest to lowest priority:
+
+* Route
+* ListenerSet
+* Gateway
+* `from` unset, in the consumer namespace
+* `from` unset, in the producer namespace
+
+In the event that multiple policies apply at the same level, [standard conflict resolution](/geps/gep-713#conflict-resolution) applies.
+
+### `Spec.Mode`
+
+With the ability to override `BackendTLSPolicy` based on context, users can modify the properties of the TLS handshake, such as changing validation parameters.
+However, there is no way to *disable* the `BackendTLSPolicy` entirely.
+
+The new `mode` field will enable this by allowing users to specify a `BackendTLSPolicy` that indicates "Do not send TLS".
+
+The `BackendTLSMode` will be an enum with values: `None`, `TLS`.
+The default for `Mode` will be `TLS`, to maintain backwards compatibility (and behavioral expectations).
+
+When `None` is specified, the only permitted field is `targetRefs`; all other fields must be empty.
+
+While this GEP only proposed these 2 fields, it is plausible the `Mode` could be extended for future use cases such as Mutual TLS.
+
+## Conformance Details
+
+Existing `BackendTLSPolicy` conformance tests apply policies to Services and test that the policy is used when reaching targeted services, and not used when reaching non-targeted Services.
+This same approach will be used with these new matching criteria to ensure implementations are choosing the correct policy to apply.
+In particular, scenarios where multiple applicable policies *could* apply will be tested to ensure proper precedence.
+
+#### Feature Names
+
+(Does it require separate feature(s) for mesh? Please add them if necessary)
+
+Yes, the following features will be added:
+
+1. BackendTLSOverride
+
+## Alternatives
```

Comment:
Likely want to at least mention HTTPRoute filter here

Reply:
Don't forget BackendPolicy, GEP-1282 as well.